Scripts~/windows/.export-release-zip.ps1

```diff
@@ -0,0 +1,35 @@
+param(
+	[Parameter(Mandatory)]
```

Comment:
Could we make this an optional field where we use the first result that matches "GameLift-CSharp-ServerSDK-UnityPlugin-*.zip" when it is not provided?

Reply:
I'd prefer that we're explicit with what SDK version we're using when we are releasing the plugin. We only have one release of the UnityPlugin ServerSDK currently; however, if we release a major version change that the Standalone plugin doesn't support, we wouldn't want to accidentally release that version and inconvenience customers